.github/workflows/run-wp-env/action.yml

```
@@ -39,16 +39,17 @@ runs:
     - name: Install wp-env
       shell: bash
       run: |
-          npm i -g --no-package-lock --no-save @wordpress/env@5.7.0
+          npm i -g --no-package-lock --no-save @wordpress/env@8.6.1
```

Lets have a single place to specify `wp-env` in (`package.json`)
```suggestion
          npm ci
```

---

I'll remove the file